src/components/Calculator.js

```diff
@@ -0,0 +1,82 @@
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Please remove the **eslint disable** statement. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  You just need to export it as the default code. :+1: